core-aam/index.html

```diff
@@ -5711,6 +5711,48 @@ <h4>Not Mapped</h4>
               >. Its absence not only indicates that the accessible object is not grabbed, but further defines it as not grab-able. These cases are marked as "Not mapped" without an asterisk.
             </p>
           </section>
+          <h4 id="ariaActions"><code>aria-actions</code></h4>
+          <table aria-labelledby="ariaActions">
+            <tbody>
+              <tr>
+                <th>ARIA Specification</th>
+                <td>
+                  <a class="property-reference" href="#aria-actions"><code>aria-actions</code></a>
+                </td>
+              </tr>
+              <tr>
+                <th>MSAA + IAccessible2</th>
+                <td>
+                  <span class="property"
+                    >Interface: <code>IAccessibleAction</code>, using the accessible nodes matching `aria-actions` IDREFs, if the referenced objects are in the accessibility tree</span
+                  ><br />
+                  <span class="property">Object Attribute: <code>aria-actions</code> &lt;value&gt;</span>
+                </td>
+              </tr>
+              <tr>
+                <th><abbr title="User Interface Automation">UIA</abbr></th>
+                <td>
+                  <span class="property"
+                    >Custom Property: <code>AccessibleActions</code>: element array of accessible nodes matching IDREFs, if the referenced objects are in the accessibility tree</span
+                  ><br />
+                  <span class="property">Object Attribute: <code>aria-actions</code> &lt;value&gt;</span>
```

Comment:
UIA doesn't have object attributes. Perhaps you meant this?

```suggestion
                  <span class="property">Property: <code>AriaProperties.actions</code>: &lt;value&gt;</span>
```

That said, I don't quite follow why we're exposing this. See above.

Reply:
Ah yup, I meant to do `Property: AriaProperties.actions: <value>`, thanks for catching that!

It came up in discussions with NVDA & JAWS -- it's definitely not as important as the IAccessibleActions pattern or the AccessibleActions custom property, but folks on those calls said it could be useful as a quick way to check if actions are intended to exist for a given node.

I'm not super opinionated on this one, and we can bring it up again to ask if it's needed.

Reply:
In conversations today I remembered another reason for this -- it should in theory allow for the possibility to query whether an element has actions even when those actions are not present in the DOM.

I think the issue with that approach is that authors also might not pass in `id` values when the elements they refer to are not rendered. I'm now wondering if we should treat the presence of the `aria-actions` attribute, even if set to the empty string, as meaningful or indicative of an element supporting actions even if they are not currently rendered.

WDYT? In theory perhaps the interface mapping with an empty array could also indicate that, and maybe this whole idea is too over-engineered 😅

Reply:
Do we definitely need to expose the fact that actions exist when they aren't currently rendered?

Reply:
I don't think so. What can a screen reader user begin with emty actions?

Reply:
@MarioBatusic the idea is that the actions _will_ exist when the control gains focus (or hover, usually), but may not exist before then. If a screen reader user is browsing with virtual cursor and doesn't have focus following the cursor, this mapping would make it possible to still let them know that actions exist. If they attempt to interact with those actions, the screen reader would focus on the control, thus presumably causing the actions to render.

Reply:
@jcsteh I updated the mapping to be an empty string rather than the value, as we discussed at TPAC. WDYT?

Also if anyone's opinions change on whether this is a valuable thing to have, let me know. I'm still a bit ambivalent (mostly worried that `aria-actions` with invalid ids will still get exposed, but nothing will show up even after focusing the control)